app/src/main/res/layout/fragment_card_view_sections.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:background="@color/n_60"
+    tools:context=".CardViewSectionsFragment"
+    android:orientation="vertical">
+
+    <androidx.cardview.widget.CardView
```

Comment:
card from your task 22 has better code structure, please use it here :) + make a constant in dimens.xml default_card_radius instead of using margin there

Reply:
Please create a style in styles.xml so we can only do style="@style/default_card_syle" :)